news/aspace.rst

```diff
@@ -0,0 +1,13 @@
+**Added:** None
+
+**Changed:** None
+
+**Deprecated:** None
+
+**Removed:** None
+
+**Fixed:**
+
+* Fixed issue with completing quotes and spaces.
```

Comment:
Actually this does become part of the CHANGELOG at release time, since we are using rever: https://regro.github.io/rever-docs/

Xonsh uses the same method for releases and has the following instructions: https://github.com/xonsh/xonsh/blob/master/CONTRIBUTING.rst#changelog